onadata/libs/utils/csv_import.py

```diff
@@ -489,21 +489,17 @@ def get_async_csv_submission_status(job_uuid):
         return async_status(FAILED, u'Empty job uuid')
 
     job = AsyncResult(job_uuid)
-    try:
-        # result = (job.result or job.state)
-        if job.state not in ['SUCCESS', 'FAILURE']:
-            response = async_status(celery_state_to_status(job.state))
-            if isinstance(job.info, dict):
-                response.update(job.info)
-
-            return response
+    # result = (job.result or job.state)
+    if job.state not in ['SUCCESS', 'FAILURE']:
+        response = async_status(celery_state_to_status(job.state))
+        if isinstance(job.info, dict):
+            response.update(job.info)
```

Comment:
By removing the `try/except` block, does that mean we no longer need to catch `BacklogLimitExceeded`?

Reply:
The `BacklogLimitExceeded` error that was handled here was raised by the `amqp` backend which is no longer present but I've seen the error is present in the `rpc` backend that's used by AMQP brokers. I think I'll re-add this for now and import the error from that backend instead